Ginger/GingerCoreNET/Drivers/CoreDrivers/Mobile/Appium/GenericAppiumDriver.cs

```diff
@@ -411,6 +409,7 @@ private void UIElementActionHandler(ActUIElement act)
                 {
                     case ActUIElement.eElementAction.JavaScriptClick:
                     case ActUIElement.eElementAction.Submit:
+                    case ActUIElement.eElementAction.Click:
```

Comment:
why this was added? i tested it before and it was working with Click

Reply:
for click, it was failing